app/styles/front.scss

```diff
@@ -227,13 +227,16 @@ $autre-lighter: #cce4e7;
 
 
 /* Should be removed once all buttons have been converted to native html input fields. */
-.btn-default {
+.btn-default, .btn {
```

Comment:
Or maybe that comment is outdated and should be removed altogether since we use `.btn` to style radio buttons… 🤔